In [2]:
import numpy as np
import math

In [4]:
lst=[2,4,6,8,10]
mean=np.mean(lst)
variance=sum([(n-mean)**2 for n in lst])/len(lst)
std=np.round(math.sqrt(variance),3)
print(f'Variance={variance}, standard deviation={std}')

Variance=8.0, standard deviation=2.828


The spread of data (variance) is 8 and the distance of the data points from the mean (standard deviation) is 2.828